src/legacy/core_plugins/kibana/public/management/sections/objects/lib/resolve_saved_objects.ts

```diff
@@ -126,7 +131,7 @@ async function importIndexPattern(
 async function importDocument(obj: SavedObject, doc: SavedObjectsRawDoc, overwriteAll: boolean) {
   await obj.applyESResp({
     references: doc._references || [],
-    ...doc,
+    ...cloneDeep(doc),
```

Comment:
Was this necessary due to a change in this PR, or is it just a fix/cleanup?

Reply:
This was necessary because of this PR - as `applyESResp` now errors out without creating an inconsistent search source instance (as in the old implementation), https://github.com/elastic/kibana/blob/master/src/plugins/saved_objects/public/saved_object/helpers/apply_es_resp.ts#L44 becomes a problem because this information is now lost for https://github.com/elastic/kibana/pull/59919/files/babb4d6ee332d3836ba81e213073a46b986dc913#diff-c5230517ea9644f8395ec776249ed4bbR174

I'm also fine with moving the clone into `applyESResp`

Reply:
probably fine to keep it there